### PR TITLE
Fixing readme's format

### PR DESCRIPTION
**Pull request by germa89**
Currently:

image

Desirable:

image

### CODE REVIEW

File: src/ansys/templates/python/pyansys_advanced/{{cookiecutter.__project_name_slug}}/README.rst [hunk 1]
```diff
@@ -79,7 +79,7 @@ need to follow these steps:
 
     .. code:: bash
 
-        git clone {{ cookiecutter.repository_url }}
+      git clone {{ cookiecutter.repository_url }}
```
Comment (RobPasMue) [inline src/ansys/templates/python/pyansys_advanced/{{cookiecutter.__project_name_slug}}/README.rst:83]: I am guessing you will need to change more lines right? =)

Comment (germa89) [reply src/ansys/templates/python/pyansys_advanced/{{cookiecutter.__project_name_slug}}/README.rst:83]: Sure!

But I like to have the PR opened first (for the screenshots purposes and such)

**Comment by germa89** [comment]
Ready to review! @RobPasMue

**Review by RobPasMue — APPROVED** [review]
LGTM!

**Pull request merged**